src/components/modal/tildel-veileder/tildel-veileder.tsx

```diff
@@ -85,13 +95,33 @@ function TildelVeileder({oversiktType, closeInput}: TildelVeilederProps) {
                     bruker.arbeidsliste.navkontorForArbeidsliste !== enhet
             );
 
-            setFnrArbeidslisteBlirSlettet(
-                fnrBrukereArbeidslisteVilBliSlettet.map(bruker => ({
-                    brukerFnr: bruker.fnr
-                }))
+            const fnrBrukereHuskelappVilBliSlettet = valgteBrukere.filter(
+                bruker =>
+                    !!bruker.huskelapp &&
+                    (bruker.veilederId !== ident || bruker.veilederId === null) &&
```

Comment:
Kan vi trekke ut `(bruker.veilederId !== ident || bruker.veilederId === null)` til ein konstant og kalle den for eksempel `tildelesNyVeileder`